spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -72,6 +67,24 @@ def run
       return self
     end
 
+    def load_cookies(content)
+      # When Ruby 2.5 support is dropped, we can safely get rid of the latter branch.
+      if YAML.name == 'Psych' && Gem::Version.new(Psych::VERSION) >= Gem::Version.new('3.1')
```

Comment:
The idea comes from https://github.com/sparklemotion/http-cookie/pull/34/files#diff-1ca99fa7d530d27cd9c91bc6cf9e1eb5da1171af6bd1dd194349de4f6f5c994cR77

Reply:
Maybe we can include it in the code as a comment as a reference?

Reply:
Hmm not really? There's no explanation for this logic there. I can add another comment on what this comparison means if you want?

Reply:
I've read the diff to understand the reasoning with Psych lib etc but I leave it to your criteria😁